pkg/doc/did/doc.go

```diff
@@ -428,3 +470,25 @@ func populateRawAuthentications(vms []VerificationMethod) []interface{} {
 
 	return rawAuthentications
 }
+
+func populateRawProofs(proofs []Proof) []interface{} {
+	var rawProofs []interface{}
+	var local Proof
+	for _, p := range proofs {
+		local = p
```

Comment:
Why? remove line `var local Proof` and `local = p` and pass data to populateRawProof without a pointer, and set capacity to `var rawProofs []interface{}` -> rawProofs := make([]interface{},0,len(proofs))

Reply:
Same as above, artificial code to avoid linter warning. I guess I can add nolint.

Reply:
Added nolint to populateRawProof which resolved this issue too.